files/default/slurm/templates/slurm_parallelcluster_partitions.conf

```diff
@@ -0,0 +1,9 @@
+# slurm_parallelcluster_partitions.conf is managed by the pcluster processes.
```

Comment:
wondering if we should rename this file to `slurm_parallelcluster.conf` or something that is not partition specific since we decided to include additional config options in it.